app/vmselect/netstorage/netstorage.go

```diff
@@ -979,7 +1004,34 @@ func LabelValues(qt *querytracer.Tracer, denyPartialResponse bool, labelName str
 	if deadline.Exceeded() {
 		return nil, false, fmt.Errorf("timeout exceeded before starting the query processing: %s", deadline.String())
 	}
-	requestData := sq.Marshal(nil)
+
+	if sq.IsMultiTenant && isTenancyLabel(labelName) {
+		tenants, err := Tenants(qt, sq.GetTimeRange(), deadline)
+		if err != nil {
+			return nil, false, err
+		}
+
+		var idx int
+		switch labelName {
+		case "vm_account_id":
+			idx = 0
+		case "vm_project_id":
```

Comment:
should we add default with assert:

logger.Fatalf("BUG: unexpected labeName=%q",labelName) ?